mapie/classification.py

```diff
@@ -943,15 +952,38 @@ def fit(
 
         estimator = check_estimator_classification(
             X,
-            y_enc,
+            y,
```

Comment:
Do you ever make a check for the y_enc? By that I mean, do you check that the output of the y_enc is similar or as expected given its size?

Reply:
I think it is something that must have been tested directly be sklearn non ?

Reply:
Let's hope so. I think generally, we'll need to create a refacto for everything that's directed at label encoding/missing labels.